modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/DeveloperKeysLayout/Inputs/index.js

```diff
@@ -45,7 +45,7 @@ const DeveloperKeysInputs = ({
 		licenseKeyDownloadURL,
 	} = useApplicationProvider();
 	const [dxpVersions, setDxpVersions] = useState([]);
-	const [selectedVersion, setSelectedVersion] = useState(dxpVersion);
+	const [selectedVersion, setSelectedVersion] = useState(dxpVersion || '');
```

Comment:
```suggestion
	const [selectedVersion, setSelectedVersion] = useState("");
```
I think that it would be better to initialize dxpVersion in a useEffect hook